### PR TITLE
Fix for #7 Babel6 source maps incorrect when nested

### PR DESCRIPTION
**Pull request by DanDeMicco**
Fix for #7

### CODE REVIEW

File: es6.js [hunk 1]
```diff
@@ -77,4 +84,4 @@ define(['babel', 'module'], function(babel, _module) {
             }
         }
     }
-});
+});
```
Comment (icecreamliker) [inline es6.js]: please append a newline at the end of the file

File: es6.js [hunk 2]
```diff
@@ -37,8 +37,15 @@ define(['babel', 'module'], function(babel, _module) {
 
         load: function(name, req, onload, config) {
 
-            var babelOptions = config.babel || {},
-                pluginOptions = config.es6 || {},
+            var babelOptions;
+            try {
+                babelOptions = JSON.parse(JSON.stringify(config.babel)); //deep clone the config. Was causing issues with nested requires and babel sourcemaps (#7)
```
Comment (icecreamliker) [inline es6.js]: please refine the comment

**Comment by icecreamliker** [comment]
If possible, please submit a new PR according to my comments, thx.

**Comment by DanDeMicco** [comment]
@icecreamliker made some small tweaks according to your feedback. Let me know if anything else needs changing

**Pull request merged**